sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/cosmosclient/ICosmosClient.scala

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.spark.cosmosclient
+
+import java.util
+
+trait ICosmosClient {
```

Comment:
Naming convention: In scala interfaces/traits don't get an I prefix

Reply:
updated